DecipherNow/workshops
# multimesh workshop updates

Changes:
- Updated diagrams
- Simplified long commands
- Renamed setup script and remove alias to make setup a bit less confusing
- Updated setup script to fail fast
- Updated AMI ID 

To test:
- Start AMI following readme instructions
- Use correct creds to start the mesh and confirm it comes up and you can hit the dashboard
- Start over, use **incorrect** creds to start and notice that it fails fast
